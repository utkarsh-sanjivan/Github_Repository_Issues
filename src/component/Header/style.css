.issues-page-header-wrapper {
    display: flex;
    justify-content: center;
    flex-direction: column;
    height: 120px;
    background-color: #3c4146;
    padding-left: 50px;
    text-align: left;
}

.issues-page-header-title {
    font-size: 24px;
    font-weight: 800;
    font-family: 'Roboto';
    color: #eef4fc;
    line-height: 48px;
}

.issues-page-header-sub-title {
    font-size: 16px;
    font-weight: 600;
    font-family: 'Roboto';
    color: #eef4fc;
    line-height: 32px;
}

.header-back-icon {
    font-size: 15px;
    color: #fff;
    margin-right: 5px;
    cursor: pointer;
}