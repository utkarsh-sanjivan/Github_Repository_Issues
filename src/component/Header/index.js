import React from 'react';
import './style.css';

const IssueHeader = props => {
  return (
    <div className="issues-page-header-wrapper">
      <div className="issues-page-header-title">{window.location.pathname === '/'? 'Issue  Search': 'Issue List'}</div>
      <div className="issues-page-header-sub-title">Search issues from a public repository of a user</div>
    </div>
  );
}

export default IssueHeader;
