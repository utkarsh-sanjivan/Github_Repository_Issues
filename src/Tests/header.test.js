import React from 'react';
import { create } from "react-test-renderer";
import Header from '../component/Header/index';

test('Test Header Component', () => {
  const component = create(<Header />);
  expect(component.toJSON()).toMatchSnapshot()
});