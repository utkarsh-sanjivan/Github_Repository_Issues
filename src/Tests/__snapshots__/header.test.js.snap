// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Header Component 1`] = `
<div
  className="issues-page-header-wrapper"
>
  <div
    className="issues-page-header-title"
  >
    Issue  Search
  </div>
  <div
    className="issues-page-header-sub-title"
  >
    Search issues from a public repository of a user
  </div>
</div>
`;
